### ISSUE TITLE
ReferenceError: Buffer is not defined 

### ISSUE THREAD

**Issue by vikil143** [zmitton/eth-proof#20]
Hey I m using this package with react-native and i am facing this problem when i import something from this package like this
`import {GetProof} from 'eth-proof';`
 it throw error is it this package was supported for react-native

**Comment by zmitton** [comment]
just use `const { GetAndVerify, GetProof, VerifyProof } = require('eth-proof')
` as it says in the readme

**Comment by vikil143** [comment]
> just use `const { GetProof } = require('eth-proof') ` as it says in the readme
Yes i import like above but it still gives an error of Buffer is not defined

**Comment by zmitton** [comment]
Oh I see. This is a dependency issue with `Buffer` I forgot to include that with my package.json file

**Comment by zmitton** [comment]
[fixed](https://github.com/zmitton/eth-proof/commit/e9ec70662a1be7a1fed87d0e150e7f501a9144ae) 

please pull latest version (2.1.5) from npm

**Comment by vikil143** [comment]
hey it still gives same error when i install latest version

**Comment by vikil143** [comment]
@zmitton  hi please help me how to tackle issue

**Comment by zmitton** [comment]
All my dependencies seem to be correct

The only thing I can find is this issue: https://github.com/feross/safe-buffer/pull/10 but they claim it was fixed, but I dont understand how it could be fixed from this pull request https://github.com/feross/safe-buffer/pull/12

I added a comment, we will see what he says I guess

**Comment by zmitton** [comment]
otherwise, if you could get a better stack trace it might be helpful

**Comment by calvinmetcalf** [comment]
are you on webpack 5 per-chance ?
